dev.Dockerfile

```diff
@@ -1,16 +1,30 @@
-FROM ruby:2.6.6
+#############
+## Stage 1 ##
+#############
+FROM ruby:2.6.6-alpine3.12 as gembuilder
 
-RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
-    echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list && \
-    curl -sL https://deb.nodesource.com/setup_14.x | bash - && \
-    apt-get install -y cmake
+RUN apk add --no-cache --update build-base cmake
```

Comment:
Why `--no-cache`?

Reply:
I guess now that it's stage 1 it matters less, but it still isn't hurting anything really (just using less space).

Reply:
Ah, so that says "don't cache things" rather than "don't use the cache"?

Reply:
AFAIK yes it's space saving.